gn/chip/tools.gni

```diff
@@ -14,5 +14,5 @@
 
 declare_args() {
   # Build CHIP tools.
-  chip_build_tools = current_os != "freertos"
+  chip_build_tools = current_os != "freertos" && current_os != "android"
```

Comment:
this could as well be accomplished with an arg, as darwin builds do

Reply:
what are your thoughts, @mspang

Reply:
This is just a default value, you can still set it via args. Setting platform-specific defaults it make configuring on the command line easier, in most cases you only need to specify CPU & OS.